docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.8'
+
+services:
+  redis:
+    image: redis:7.0.0-alpine
+    command: redis-server
+    ports:
+      - 6379:6379
```

Comment:
It's fine to have a docker compose file for testing, but we need to update the `package.json` to start docker in development.

Here is the command that can be used:

```
"dev:docker": "docker compose -p ${npm_package_name} up",
```

You should also update the `npm run dev` command to start also the `dev:docker`

Reply:
Why `${npm_package_name}`?